integrations/gaia/README.md

```diff
@@ -0,0 +1,149 @@
+  throw new Error("WALLET_KEY must be set");
+}
+
+if (!ENCRYPTION_KEY) {
+  throw new Error("ENCRYPTION_KEY must be set");
+}
+
+/* Check if the Gaia API key is set */
+if (!GAIA_API_KEY) {
+  throw new Error("GAIA_API_KEY must be set");
+}
+
+/* Check if the Gaia node's base URL is set */
+if (!GAIA_NODE_URL) {
+  throw new Error("GAIA_NODE_URL must be set");
+}
+
+/* Check if the the model name for the Gaia node is set */
+if (!GAIA_MODEL_NAME) {
+  throw new Error("GAIA_MODEL_NAME must be set");
+}
+
+const signer = createSigner(WALLET_KEY);
+const encryptionKey = getEncryptionKeyFromHex(ENCRYPTION_KEY);
+const openai = new OpenAI({ 
+    baseURL: GAIA_NODE_URL,
+    apiKey: GAIA_API_KEY
+});
+
+/* Set the environment to dev or production */
+const env: XmtpEnv = "dev";
+
+async function main() {
+  console.log(`Creating client on the '${env}' network...`);
+  const client = await Client.create(signer, encryptionKey, {
+    env,
+  });
+
+  console.log("Syncing conversations...");
+  /* Sync the conversations from the network to update the local db */
+  await client.conversations.sync();
+
+  console.log(
+    `Agent initialized on ${client.accountAddress}\nSend a message on http://xmtp.chat/dm/${client.accountAddress}`,
+  );
+
+  console.log("Waiting for messages...");
+  const stream = client.conversations.streamAllMessages();
+
+  for await (const message of await stream) {
+    /* Ignore messages from the same agent or non-text messages */
+    if (
+      message?.senderInboxId.toLowerCase() === client.inboxId.toLowerCase() ||
+      message?.contentType?.typeId !== "text"
+    ) {
+      continue;
+    }
+
+    console.log(
+      `Received message: ${message.content as string} by ${message.senderInboxId}`,
+    );
+
+    const conversation = client.conversations.getConversationById(
+      message.conversationId,
+    );
+
+    if (!conversation) {
+      console.log("Unable to find conversation, skipping");
+      continue;
+    }
+
+    try {
+      const completion = await openai.chat.completions.create({
+        messages: [{ role: "user", content: message.content as string }],
+        model: GAIA_MODEL_NAME,
+      });
+
+      /* Get the AI response */
+      const response =
+        completion.choices[0]?.message?.content ||
+        "I'm not sure how to respond to that.";
+
+      console.log(`Sending AI response: ${response}`);
+      await conversation.send(response);
+    } catch (error) {
+      console.error("Error getting AI response:", error);
+      await conversation.send(
+        "Sorry, I encountered an error processing your message.",
+      );
+    }
+
+    console.log("Waiting for messages...");
+  }
+}
+
+main().catch(console.error);
+```
+
+## Run the agent
+
+```bash
+# git clone repo
+git clone https://github.com/ephemeraHQ/xmtp-agent-examples.git
+# go to the folder
+cd xmtp-agent-examples
+# install packages
+yarn
+# generate random keys (optional)
+yarn gen:keys
+# run the example
+yarn examples:gpt
```

Comment:
please update this to:

```
cd integrations/gaia
yarn dev
```

Reply:
Done. Updated in f6e55a3d26b9a32bdabee6b04dd08541f38b7093